postgres-protocol/src/message/backend.rs

```diff
@@ -435,7 +435,7 @@ impl BackendKeyDataBody {
 }
 
 pub struct CommandCompleteBody {
-    tag: Bytes,
+    pub tag: Bytes,
```

Comment:
nit: more in line with the style of these structs to define "pub fn into_bytes()" instead of making tag public.

Reply:
kk, added tag_bytes for &Bytes that doesn't consume CommandCompleteBody and into_bytes for owned  Bytes that does consume it.